src/language-js/print/template-literal.js

```diff
@@ -14,11 +14,13 @@ const {
     addAlignmentToDoc,
   },
   printer: { printDocToString },
+  utils: { mapDoc },
 } = require("../../document");
 const {
   isBinaryish,
   isJestEachTemplateLiteral,
   isSimpleTemplateLiteral,
+  uncook,
```

Comment:
Let's move `uncook` to this file too. It's strictly related to printing template literals.